### ISSUE TITLE
Release v1.30.6

### ISSUE THREAD

**Issue by deckhouse-BOaTswain** [deckhouse/deckhouse#733]
:point_right: Use this issue to test milestone [v1.30.6](https://github.com/deckhouse/deckhouse/milestone/13) and deploy released tag.
            :point_right: Use 'e2e/run/' labels to run default e2e test.
            :point_right: Use 'e2e/use/' labels to run specific e2e test.
            :point_right: Use 'deploy/' labels to deploy site and documentation.
            :point_right: Use 'deploy/deckhouse/' labels to deploy to channels after creating tag.

**Comment by deckhouse-BOaTswain** [comment]
New commit [617e90](https://github.com/deckhouse/deckhouse/commit/617e907f6d4e2fbaeef18f3935b857453dfc51bb) in branch 'main':
```
Re-generate changelog (#734)

Co-authored-by: konstantin-axenov <konstantin-axenov@users.noreply.github.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New tag 'v1.30.6' is created.

**Comment by deckhouse-BOaTswain** [comment]
New commit [a1c583](https://github.com/deckhouse/deckhouse/commit/a1c583f1d721eaf972c67797984ad611e622c769) in branch 'main':
```
[core][dhctl] Fix potential panic for bashible logs in `dhctl bootstrap` command (#724)
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [99a573](https://github.com/deckhouse/deckhouse/commit/99a5734b8219a1b6ee25234fe73e4a75d92e4fbc) in branch 'main':
```
[istio] canary usage doc fix (#731)

[istio] Canary usage doc fix
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [9bf80d](https://github.com/deckhouse/deckhouse/commit/9bf80db65400c34ef54dd97a516dc4cccc3854eb) in branch 'main':
```
[ci] fix: proper image tag name for pull_request_target images (#740)
```
